azerothcore/azerothcore-wotlk
# feat(Config): Max gold on change faction

## Changes Proposed:
-  Make the code configurable, currently there were no way to do it without recompiling it.
-  Disabled by default.

## Issues Addressed:

- Closes none addressed.

## SOURCE:


## Tests Performed:

- Tested in-game with multiple values.

## How to Test the Changes:


1. With the config set to 0, you shouldn't have any limit for the character to do the faction change.
2. With 1000, you should be able to have only 10 silver on your bags when doing the faction change.
3. Set different values for the config and do a .reload config on console.
